addons/crm/report/crm_activity_report_views.xml

```diff
@@ -87,7 +87,6 @@
                 'graph_mode': 'bar',
                 'graph_groupbys': ['date:month', 'subtype_id'],
             }</field>
-           <field name="domain">[('user_id', '=', uid)]</field>
```

Comment:
Actually when targeting stable you should set domain to False. Otherwise when updating the view, as there is no new value the old domain will stay.

Reply:
done